templates/server-route.yaml

```diff
@@ -14,20 +15,25 @@ metadata:
     app.kubernetes.io/name: {{ include "vault.name" . }}
     app.kubernetes.io/instance: {{ .Release.Name }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
-    {{- with .Values.server.route.labels }}
+    {{- with $route.labels }}

```

Comment:
I dont see this function in helper.tpl

Reply:
Its not a function. Its a variable defined at the top of the file taking value from Values.server.route

Reply:
ok. thanks for explanation